### PR TITLE
Update manage-folders-and-rules-feature.md

### LINKED ISSUE #12982: Some examples would be good
This page would be useful if some examples were given on how to use the commands detailed. Otherwise - how do you use them?

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 62ab7809-9202-5fce-7b0a-ff090a920ce5
* Version Independent ID: 62ab7809-9202-5fce-7b0a-ff090a920ce5
* Content: [Manage Folders and Rules feature in Microsoft 365 Groups - Microsoft 365 Enterprise](https://learn.microsoft.com/en-us/microsoft-365/enterprise/manage-folders-and-rules-feature?view=o365-worldwide)
* Content Source: [microsoft-365/enterprise/manage-folders-and-rules-feature.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/enterprise/manage-folders-and-rules-feature.md)
* Service: **o365-administration**
* GitHub Login: @MikePlumleyMSFT
* Microsoft Alias: **mikeplum**

### PR DESCRIPTION
**Pull request by msbemba**
Added examples

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12982

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/enterprise/manage-folders-and-rules-feature.md [hunk 1]
```diff
@@ -49,6 +49,13 @@ Admin can enable the feature with the help of cmdlet `Set-OrganizationConfig -Is
...
   > - Message level actions Copy/Move will be blocked.
 
+### Example 1
+To enable folders and Rules feature for Microsoft 365 Groups in Outlook you can use the below cmdlet
+
+```powershell
+Set-OrganizationConfig -IsGroupFoldersAndRulesEnabled $true
+```
+
 Once the feature is enabled, by default only owner of the group has the permission to create folder, rename folder, move and copy messages across folders.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/manage-folders-and-rules-feature.md:59]:
```suggestion

```powershell
Set-OrganizationConfig -IsGroupFoldersAndRulesEnabled $true
```
This example enables the folders and rules feature for Microsoft 365 Groups in Outlook.

Once the feature is enabled, by default, only the group's owner has permission to create folders, rename folders, and move and copy messages across folders.
```

File: microsoft-365/enterprise/manage-folders-and-rules-feature.md [hunk 2]
```diff
@@ -67,6 +74,14 @@ Admin can enable the member permission to the tenant using the cmdlet `IsGroupMe
 
    Default value: false
 
+### Example 1
+
+To enable Enable member permission option you can use the below cmdlet.
+
+```powershell
+Set-OrganizationConfig -IsGroupMemberAllowedToEditContent $true
+```
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/manage-folders-and-rules-feature.md:83]:
```suggestion

To enable the Enable member permission option, you can use the following cmdlet.

```powershell
Set-OrganizationConfig -IsGroupMemberAllowedToEditContent $true
```
```

File: microsoft-365/enterprise/manage-folders-and-rules-feature.md [hunk 3]
```diff
@@ -49,6 +49,13 @@ Admin can enable the feature with the help of cmdlet `Set-OrganizationConfig -Is
   > 
   > - Message level actions Copy/Move will be blocked.
 
+### Example 1
+To enable folders and Rules feature for Microsoft 365 Groups in Outlook you can use the below cmdlet
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/manage-folders-and-rules-feature.md]:
```suggestion
To enable the **Folders and Rules** feature for Microsoft 365 Groups in Outlook, you can use the following cmdlet:
```

File: microsoft-365/enterprise/manage-folders-and-rules-feature.md (hunk 1 above)
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/manage-folders-and-rules-feature.md]:
```suggestion
Once the feature is enabled, by default, only the group owner has permission to create folders, rename folders, and move and copy messages across folders.
```

File: microsoft-365/enterprise/manage-folders-and-rules-feature.md [hunk 4]
```diff
@@ -67,6 +74,14 @@ Admin can enable the member permission to the tenant using the cmdlet `IsGroupMe
 
    Default value: false
 
+### Example 1
+
+To enable Enable member permission option you can use the below cmdlet.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/manage-folders-and-rules-feature.md]:
```suggestion
To enable the **Enable member permission** option, you can use the following cmdlet:
```

File: microsoft-365/enterprise/manage-folders-and-rules-feature.md [hunk 5]
```diff
@@ -67,6 +74,14 @@ Admin can enable the member permission to the tenant using the cmdlet `IsGroupMe
...
+
+```powershell
+Set-OrganizationConfig -IsGroupMemberAllowedToEditContent $true
+```
+
 Once this is enabled, group owners can provide group members with the ability to create folders, rename folders, copy, move, and delete messages by navigating to the group from Outlook > **Settings** > **Edit Group** > and selecting the option **All members will be able to create, edit, move, copy, and delete mail folders and rules within the group**. Group level member permission is handled by group owners.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/manage-folders-and-rules-feature.md]:
```suggestion
Once this option is enabled, group owners can provide group members with the ability to create folders, rename folders, and copy, move, and delete messages by navigating to the group from Outlook > **Settings** > **Edit Group** > and selecting the option **All members will be able to create, edit, move, copy, and delete mail folders and rules within the group**. Group-level member permission is handled by group owners.
```

File: microsoft-365/enterprise/manage-folders-and-rules-feature.md [hunk 6]
```diff
@@ -84,6 +99,14 @@ Admins can block the **Move** message option for all Microsoft 365 groups within
...
+### Example 1
+
+To block “Move” message capability you can use the below cmdlet
+
+```powershell
+Set-OrganizationConfig -BlockMoveMessagesForGroupFolders $true
+```
+
 > [!NOTE]
 > Creation of **Move** rule is also disabled when `BlockMoveMessagesForGroupFolders` is enabled.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/manage-folders-and-rules-feature.md]:
```suggestion
> Creation of the **Move** rule is also disabled when `BlockMoveMessagesForGroupFolders` is enabled.
```

File: microsoft-365/enterprise/manage-folders-and-rules-feature.md [hunk 7]
```diff
@@ -84,6 +99,14 @@ Admins can block the **Move** message option for all Microsoft 365 groups within
 
    Default value: false
 
+### Example 1
+
+To block “Move” message capability you can use the below cmdlet
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/manage-folders-and-rules-feature.md]:
```suggestion
To block the **Move message** capability, you can use the following cmdlet:
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@MikePlumleyMSFT Please merge this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**